tests/test_six_calendar.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+import pandas as pd
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.exchange_calendar_six import SIXExchangeCalendar
+
+
+class LSECalendarTestCase(ExchangeCalendarTestBase, TestCase):
```

Comment:
Noticed a copy/paste error - should be `SIXCalendarTestCase`

Reply:
sorry, fixed